From e2f3f3eae480e8b484f04487167ebdb429622049 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 20 Nov 2017 14:19:53 +0800
Subject: [PATCH 2835/5242] MLK-16902 ARM64: dts: Update the default critical
 temp setting on imx8mq

commit  fc9c2512a1670329a5fc2ea4d28caa0e8b73991e from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX8MQ silicon only has consumer and industrial part, for consumer part,
the operating temperature range is 0C ~ 95C, so update the thermal driver
critical temp to 95C. This is only a reference setting based on consumer
part, if the silicon is for industrial use, please update the temperature
according to the industrial part datasheet.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 96a3803..f1a7aae 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -430,7 +430,7 @@
 					type = "passive";
 				};
 				cpu_crit0: trip1 {
-					temperature = <125000>;
+					temperature = <95000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
-- 
1.7.9.5

